### PR TITLE
USWDS - Docs: Add initial Key Benefits USWDS page.

### PR DESCRIPTION
**Pull request by mejiaj**
## Description

[Preview link](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/jm-why-uswds/about/key-benefits/)

Closes https://github.com/uswds/uswds/issues/4150

Before you hit Submit, make sure you’ve done whichever of these applies to you:

- [ ] Follow the [18F Front End Coding Style Guide](https://pages.18f.gov/frontend/) and [Accessibility Guide](https://pages.18f.gov/accessibility/checklist/).
- [ ] Run `npm test` and make sure the tests for the files you have changed have passed.
- [ ] Run your code through [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/) and make sure it’s error free.
- [x] Title your pull request using this format: [Website] - [UI component]: Brief statement describing what this pull request solves.

### CODE REVIEW

**Comment by KMullan944** [comment]
This looks great! Just a few edits, per below.

- In that intro section where we list the 6 benefits we expound upon in sections below, could you add jump-links to those sections? Or is that too distracting and was it omitted as a design decision?
- In "Cross-functional design system community" section, there's an external link to Github that has an icon with it. Pls delete.
- In the footer (above the Identifier?), "Become part of the community," there are 2 external links that need icon removed: Github and Twitter

And I think that's it! It reads well and is clear -- those are my only recommended edits. Thank you :)

**Comment by mejiaj** [comment]
@KMullan944 thank you for the feedback! The external links will be updated on a separate pull request (#1204).

Yes, I'll go ahead and update the list so it links below.

**Review by KMullan944 — APPROVED** [review]

File: pages/about/key-benefits.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+      <div class="usa-icon-list__icon text-blue">
+        <svg class="usa-icon" aria-hidden="true" role="img">
+          <use xlink:href="{{ site.baseurl }}/assets/img/sprite.svg#check_circle"></use>
+        </svg>
+      </div>
+      <div class="usa-icon-list__content">
+        <p><b>Use these key benefits listed to make the case to your team and management.</b> You may customize the themes and messages for your audience, and email us at <a href="mailto:uswds@support.digitalgov.gov">uswds@support.digitalgov.gov</a> if you have feedback on things to add or remove from this list.</p>
+      </div>
+    </li>
+    <li class="usa-icon-list__item margin-bottom-2">
```
Comment (thisisdano) [inline pages/about/key-benefits.md:106]: Remove custom margin

```suggestion
    <li class="usa-icon-list__item">
```

File: pages/about/key-benefits.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+<div class="margin-top-7 border-05 border-primary-light bg-primary-lighter padding-3">
+
+  <h2 id="how-to-make-the-case" class="margin-top-0">How to make the case for USWDS</h2>
+
+  <p>
+    We know that making the case for USWDS is important and sometimes difficult. Here are a few tips!
+  </p>
+
+  <ul class="usa-icon-list usa-icon-list--size-lg margin-top-5">
+    <li class="usa-icon-list__item margin-bottom-2">
```
Comment (thisisdano) [inline pages/about/key-benefits.md:96]: Remove custom margin

```suggestion
    <li class="usa-icon-list__item">
```

File: pages/about/key-benefits.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+We are as committed as you are to public service, and we reflect this in our product, community, and customer support.
+
+
+
+<div class="margin-top-7 border-05 border-primary-light bg-primary-lighter padding-3">
+
+  <h2 id="how-to-make-the-case" class="margin-top-0">How to make the case for USWDS</h2>
+
+  <p>
```
Comment (thisisdano) [inline pages/about/key-benefits.md:91]: Reduce top margin

```suggestion
  <p class="margin-top-1">
```

File: pages/about/key-benefits.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+This process helps get the most from the teams that use the design system. It builds the institutional memory available to new teams. It promotes user-centered, data-driven  decision making and governance. It results in faster, more effective service delivery over time. Everything we invest in the design system is shared, multiplied, and compounded to the greatest extent possible. We use the scale of the federal government to help individual teams perform at their peak.
+
+We are as committed as you are to public service, and we reflect this in our product, community, and customer support.
+
+
+
+<div class="margin-top-7 border-05 border-primary-light bg-primary-lighter padding-3">
+
+  <h2 id="how-to-make-the-case" class="margin-top-0">How to make the case for USWDS</h2>
```
Comment (thisisdano) [inline pages/about/key-benefits.md:89]: Remove margin

```suggestion
  <h2 id="how-to-make-the-case" class="margin-x-0">How to make the case for USWDS</h2>
```

File: pages/about/key-benefits.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+
+USWDS exists to scale service expertise across the federal government, so all teams can benefit from any team's improvements. When you build with USWDS, you benefit from the experience of teams and individuals from your agency and others. What you build helps improve the design system for the next teams that use it.
+
+This process helps get the most from the teams that use the design system. It builds the institutional memory available to new teams. It promotes user-centered, data-driven  decision making and governance. It results in faster, more effective service delivery over time. Everything we invest in the design system is shared, multiplied, and compounded to the greatest extent possible. We use the scale of the federal government to help individual teams perform at their peak.
+
+We are as committed as you are to public service, and we reflect this in our product, community, and customer support.
+
+
+
+<div class="margin-top-7 border-05 border-primary-light bg-primary-lighter padding-3">
```
Comment (thisisdano) [inline pages/about/key-benefits.md:87]: Tighten padding on mobile

```suggestion
<div class="margin-top-7 border-05 border-primary-light bg-primary-lighter padding-2 tablet:padding-3">
```

File: pages/about/key-benefits.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+
+  <h2 id="how-to-make-the-case" class="margin-top-0">How to make the case for USWDS</h2>
+
+  <p>
+    We know that making the case for USWDS is important and sometimes difficult. Here are a few tips!
+  </p>
+
+  <ul class="usa-icon-list usa-icon-list--size-lg margin-top-5">
```
Comment (thisisdano) [inline pages/about/key-benefits.md:95]: Reduce top margin

```suggestion
  <ul class="usa-icon-list usa-icon-list--size-lg margin-top-3">
```

**Review by thisisdano — CHANGES_REQUESTED** [review]

File: pages/about/key-benefits.md (hunk 4 above)
Comment (mejiaj) [reply pages/about/key-benefits.md:89]: Should this be `margin-y-0`?

**Review by thisisdano — APPROVED** [review]

**Pull request merged**